tuxemon/core/states/world/worldstate.py

```diff
@@ -622,6 +622,8 @@ def get_exits(self, position, collision_map=None, skip_nodes=None):
         tile_data = collision_map.get(position)
         if tile_data:
             exits = self.get_explicit_tile_exits(position, tile_data, skip_nodes)
+        else:
+            exits = None
```

Comment:
`exits` is checked in the loop below. When it isn't set to a useful value, it must still be set to None, as Python crashes if you reference a variable that hasn't been initialized. It's very silly that I missed this one.